File: src/test/java/test/CreateOrganizationTest.java
import cn.authing.sdk.java.client.ManagementClient;
import cn.authing.sdk.java.dto.*;
import cn.authing.sdk.java.model.ManagementClientOptions;
import cn.authing.sdk.java.util.JsonUtils;

import java.util.Collections;



public class CreateOrganizationTest {

    private static final String ACCESS_KEY_ID = "YOUR_ACCESS_KEY_ID";
    private static final String ACCESS_KEY_SECRET = "YOUR_ACCESS_KEY_SECRET";


    public static void main(String[] args) throws Throwable {
        ManagementClientOptions clientOptions = new ManagementClientOptions(ACCESS_KEY_ID, ACCESS_KEY_SECRET);
        ManagementClient client = new ManagementClient(clientOptions);

        CreateOrganizationReqDto request = new CreateOrganizationReqDto();
        request.setOrganizationName("organizationName_7021");
        request.setOrganizationCode("organizationCode_8252");
        request.setDescription("description_1220");
        request.setOpenDepartmentId("openDepartmentId_1290");
        request.setI18n(new cn.authing.sdk.java.dto.OrganizationNameI18nDto());

        OrganizationSingleRespDto response = client.createOrganization(request);
        System.out.println(JsonUtils.serialize(response));
    }

}

File: src/test/go/test/CreateOrganizationTest.go
package main

import (
"authing-go-sdk/client"
"authing-go-sdk/dto"
"fmt"
)

func main() {
    options := client.ManagementClientOptions{
        AccessKeyId:     "AUTHING_USERPOOL_ID",
        AccessKeySecret: "AUTHING_USERPOOL_SECRET",
    }
    var err error
    client, err := client.NewClient(&options)
    if err != nil {
        panic(err)
    }
    request := dto.CreateOrganizationReqDto{
        OrganizationName: "organizationName_2144",
        OrganizationCode: "organizationCode_5038",
        Description: "description_9809",
        OpenDepartmentId: "openDepartmentId_6601",
        I18n: nil,
    }
    response := client.CreateOrganization(&request)
    fmt.Println(response)
}

